### PR TITLE
Fix attention mask for padded inputs

### PR DESCRIPTION
**Pull request by adrianeboyd**
Fix `RaggedArray.to_padded()` to pad using the provided value.

In `TransformersWrapper.get_model_kwargs()`, pad with `-1` in order to be able to create the correct `attention_mask`.

Addresses inconsistent results due to batching reported in #132 and #135.

### CODE REVIEW

**Comment by adrianeboyd** [comment]
For reference, a test script combining cases analyzed in #132 and #135:

```
from spacy_transformers.language import TransformersLanguage
from spacy_transformers.pipeline.wordpiecer import TransformersWordPiecer
from spacy_transformers.pipeline.tok2vec import TransformersTok2Vec
import spacy
import numpy as np
import cupy

texts = [
    "Hello my name is daniel",
    "I would like to participate in interesting events",
    "something is very strange with SpaCy",
    "The vectors aren't similar",
    "let us try to round both vectors to 5 decimals",
    "let us try to round both vectors to 5 decimals " * 25,
]

def run(model):
    print("evaluating model:", model, "...", end=" ")
    if "_" in model:
        nlp = spacy.load(model)
    else:
        nlp = TransformersLanguage(trf_name=model)

        wordpiecer_pipe = TransformersWordPiecer.from_pretrained(nlp.vocab, model)
        tok2vec_pipe = TransformersTok2Vec.from_pretrained(nlp.vocab, model)

        nlp.add_pipe(nlp.create_pipe("sentencizer"))
        nlp.add_pipe(wordpiecer_pipe)
        nlp.add_pipe(tok2vec_pipe)

    outputs1 = list(nlp.pipe(texts, batch_size=1))
    outputs2 = list(nlp.pipe(texts, batch_size=50))

    for output1, output2 in zip(outputs1, outputs2):
        assert np.allclose(output1[0]._.trf_last_hidden_state, output2[0]._.trf_last_hidden_state, rtol=1e-05, atol=1e-02)
        assert np.allclose(output1._.trf_last_hidden_state, output2._.trf_last_hidden_state, rtol=1e-05, atol=1e-02)
        assert np.allclose(cupy.asnumpy(output1.tensor), cupy.asnumpy(output2.tensor), rtol=1e-05, atol=1e-02)
    print("passed")

if __name__ == "__main__":
    for model in [
        "en_trf_robertabase_lg",
        "en_trf_bertbaseuncased_lg",
        "de_trf_bertbasecased_lg",
        "en_trf_xlnetbasecased_lg",
        "en_trf_distilbertbaseuncased_lg",
        "distilbert-base-uncased",
        "bert-base-uncased",
        "roberta-base",
        "bert-base-german-cased",
        "gpt2",
        "xlnet-base-cased",
    ]:
        run(model)
```

Note: truncation (mentioned in #135) is still an issue and this will still crash with some models given individual sentences that are too long.

**Comment by honnibal** [comment]
Ah, thanks! The masks are always giving me trouble...
